datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml

```diff
@@ -190,10 +190,6 @@
   description: |
     If `tls_verify` is disabled, security warnings are logged by the check.
     Disable those by setting `tls_ignore_warning` to true.
-
-    Note: `tls_ignore_warning` set to true is currently only reliable if used by one instance of one integration. 
-    If enabled for multiple instances, spurious warnings might still appear even if `tls_ignore_warning` is set
-    to true.
```

Comment:
It looks like your http.yaml is not the latest, perhaps you may have copy pasted an older file? This is still being removed because the latest from master is not on your branch:
https://github.com/DataDog/integrations-core/blob/87e6f39df8981fc73ea3fe741c60aed467aa2581/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L194-L196

The easiest solution would be to copy-paste this note back to this file to sync it up as it seems to be the main diff/unintentional change.